tools/generate-identifier-regex.js

```diff
@@ -10,42 +10,29 @@ var get = function(what) {
     return require('unicode-' + version + '/' + what + '/code-points');
 };
 
-// Unicode categories needed to construct the ES5 regex
-var Lu = get('categories/Lu');
-var Ll = get('categories/Ll');
-var Lt = get('categories/Lt');
-var Lm = get('categories/Lm');
-var Lo = get('categories/Lo');
-var Nl = get('categories/Nl');
-var Mn = get('categories/Mn');
-var Mc = get('categories/Mc');
-var Nd = get('categories/Nd');
-var Pc = get('categories/Pc');
-
-var generateES5Regex = function() { // ES 5.1
-    // http://mathiasbynens.be/notes/javascript-identifiers#valid-identifier-names
-    var identifierStart = regenerate('$', '_')
-        .add(Lu, Ll, Lt, Lm, Lo, Nl)
-        .removeRange(0x010000, 0x10FFFF) // remove astral symbols
+var generateES6Regex = function() { // ES 6
+    // https://mathiasbynens.be/notes/javascript-identifiers-es6
+    var identifierStart = regenerate(get('properties/ID_Start'))
```

Comment:
Maybe add `.add('$', '_')` here for clarity? They’ll get removed anyway but it just makes the code match the spec more closely.